ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -2403,6 +2466,7 @@ DXZC:
   - overload_compressor_alarm
   - high_discharge_air_temperature_alarm
   - low_discharge_air_temperature_alarm
+  - cooling_request_count
```

Comment:
Please confrim that this is a single zone unit if it has cooling requests. Where are the multiple requests coming from?